packages/components/Button/src/Button.types.ts

```diff
@@ -9,9 +9,10 @@ import { IViewProps } from '@fluentui-react-native/adapters';
 
 export const buttonName = 'Button';
 export type ButtonSize = 'small' | 'medium' | 'large';
-export type ButtonAppearance = 'primary' | 'subtle';
+export type ButtonAppearance = 'primary' | 'subtle' | 'ascent' | 'outline';
```

Comment:
SPEC should be updated for this

Reply:
Also the addition of outline to ButtonCoreTokens and the updated comment to appearance